contrib/Newrelic/Exporter.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\Newrelic;
+
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Middleware;
+use GuzzleHttp\Psr7\Request;
+use Http\Adapter\Guzzle7\Client;
+use InvalidArgumentException;
+use OpenTelemetry\Sdk\Trace;
+use OpenTelemetry\Trace as API;
+use Psr\Http\Client\ClientExceptionInterface;
+use Psr\Http\Client\ClientInterface;
+use Psr\Http\Client\NetworkExceptionInterface;
+use Psr\Http\Client\RequestExceptionInterface;
+
+/**
+ * Class NewrelicExporter - implements the export interface for data transfer via Newrelic protocol
+ * @package OpenTelemetry\Exporter
+ *
+ * This is an experimental, non-supported exporter.
+ * It will send PHP Otel trace data end to end across the internet to a functional backend.
+ * Needs a license key to connect.  For a free account/key, go to: https://newrelic.com/signup/
+ */
+class Exporter implements Trace\Exporter
+{
+    /**
+     * @var string
+     */
+    private $endpointUrl;
+
+    /**
+     * @var string
+     */
+    private $licenseKey;
+
+    /**
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @var SpanConverter
+     */
+    private $spanConverter;
+
+    /**
+     * @var bool
+     */
+    private $running = true;
+
+    /**
+     * @var ClientInterface
+     */
+    private $client;
+
+    public function __construct(
+        $name,
+        string $endpointUrl,
+        string $licenseKey,
+        SpanConverter $spanConverter = null,
+        ClientInterface $client = null
+    ) {
+        $parsedDsn = parse_url($endpointUrl);
+
+        if (!is_array($parsedDsn)) {
+            throw new InvalidArgumentException('Unable to parse provided DSN');
+        }
+        if (
+            !isset($parsedDsn['scheme'])
+            || !isset($parsedDsn['host'])
+            || !isset($parsedDsn['path'])
+        ) {
+            throw new InvalidArgumentException('Endpoint should have scheme, host, port and path');
+        }
+
+        $this->licenseKey = $licenseKey;
+        $this->endpointUrl = $endpointUrl;
+        $this->name = $name;
+        $this->client = $client ?? $this->createDefaultClient();
+        $this->spanConverter = $spanConverter ?? new SpanConverter($name);
+    }
+
+    /**
+     * Exports the provided Span data via the Newrelic protocol
+     *
+     * @param iterable<API\Span> $spans Array of Spans
+     * @return int return code, defined on the Exporter interface
+     */
+    public function export(iterable $spans): int
+    {
+        if (!$this->running) {
+            return Exporter::FAILED_NOT_RETRYABLE;
+        }
+
+        if (empty($spans)) {
+            return Trace\Exporter::SUCCESS;
+        }
+
+        $convertedSpans = [];
+        foreach ($spans as $span) {
+            array_push($convertedSpans, $this->spanConverter->convert($span));
+        }
+        $commonAttributes = ['attributes' => [ 'service.name' => $this->name,
```

Comment:
I think this 2 arrays($commonAttributes and $payload) needs some formatting. Maybe you can run `make style` to fix them for you.

Reply:
Thanks for commenting @beniamin! These comments apply to the #275 as well, so I will apply them there  as well :)

`make style` didn't change the formatting, so I'll format it by hand.